mp/src/game/client/hud_crosshair.cpp

```diff
@@ -327,10 +327,10 @@ void CHudCrosshair::DrawCrosshair( CWeaponBase *weaponBase )
 
     if (cl_crosshair_style.GetInt() != 2)
     {
-        m_pCrosshair = gHUD.GetIcon("whiteAdditive");
```

Comment:
If `m_pCrosshair` is no longer used you can remove the variable from this class.

Reply:
m_pCrosshair is still used for drawing the default "dots" crosshair.